components/apim/main.tf

```diff
@@ -14,11 +14,13 @@ module "api-mgmt" {
   virtual_network_resource_group = local.vnet_rg
   virtual_network_name           = local.vnet_name
   environment                    = var.env
+  env                            = var.env
   virtual_network_type           = "Internal"
   department                     = var.department
   common_tags                    = module.ctags.common_tags
   route_next_hop_in_ip_address   = local.hub[var.hub].ukSouth.next_hop_ip
   publisher_email                = var.publisher_email
+  product                        = var.product
```

Comment:
also for this - we can just set here
https://github.com/hmcts/cnp-module-api-mgmt-private/blob/390d21741e05b257fd0104aeebf96bb476ff5ede/app-insights.tf#L5 
with `product               = var.department`